tasks/functions/functions.js

```diff
@@ -52,7 +69,12 @@ function getGoogle(n) {
  * }
  */
 function getUser(firstName, lastName, age) {
```

Comment:
please try to use default function parameters here, such as 
function getUser(firstname = null, ...) {
return {
firstname,
...
}
}

Reply:
thank you very much. Fixed.)